dump-dvd-kit.py

```diff
@@ -5,6 +5,11 @@
 import usb1
 import struct
 
+REGION_LOOKUP = ["region-free", "North-America", 
+                 "Europe", "Southeast Asia", 
+                 "Latin America", "Africa", 
+                 "China", "MPAA", "International"]
```

Comment:
Are these confirmed to match the region in the dongles?
Why is one of them in all lowercase, but then regions use proper uppercase?

The full list also implies that we have encountered all of those in the real world (which likely isn't the case).
I think it's better to add a dictionary with information, or to just not print this information at all (instead, [documenting it on the wiki](https://xboxdevwiki.net/Xbox_DVD_Movie_Playback_Kit#Known_versions) where we can eventually make sense of it, once confirmed).